doc/installation.rst

```diff
@@ -1,116 +1,121 @@
 .. _installation:
 .. _Installing hgvs:
 
+
 Installing hgvs
 !!!!!!!!!!!!!!!
 
 
 Supported Platforms
 @@@@@@@@@@@@@@@@@@@
 
-`hgvs` is developed primarily on Ubuntu systems and has been reported
-to work on Mac.  Other platforms and dependency versions are expected
-to work but have not been tested. Reports of successful operation on
-other platforms (and patches to enable this) are appreciated.
-**Python 2 is required.** (Python 3 support is in progress.)
+**Operating Systems: Ubuntu, MacOS**
```

Comment:
This section isn't quite right. hgvs works on Python 2.7 and 3.5+ now, and on Ubuntu and MacOS. It doesn't work on Windows (yet). On March 31, 2019, development will switch to Python 3.6+ only; no new features or bug fixes for Python 2.7-compatible versions of hgvs will be provided.